src/browser/ui/ReactDOMComponent.js

```diff
@@ -275,6 +280,7 @@ ReactDOMComponent.Mixin = {
    * @param {object} context
    * @return {string} Content markup.
    */
+   /* eslint-enable max-len */
```

Comment:
I actually don't want any of these in here. It's pure noise and we'll forget to remove them if we change types. I do wish there was an option to eslint to ignore the long lines in comments - I would rather fork the rule and make that happen than have these in there.